electrophysiology/models.py

```diff
@@ -70,7 +80,9 @@ class SpikeSorting(BaseExperimentalData):
 
     """
     spike_times = models.ForeignKey(Dataset, blank=True, null=True, related_name="spike_sorting_spike_times",
-                                    help_text="time of each spike relative to experiment start in seconds. TODO: experiment or recording start?")
+                                    help_text="time of each spike relative to experiment start in seconds.")
+									# TODO: experiment or recording start?
+									# Experiment start. Everything should be relative to that.
```

Comment:
So they'll all start at some arbitrary time in seconds - and you should subtract the recording_start offset to get the .dat spiketimes?